supporting_content/dimensions/pricingmeasure.md

```diff
@@ -0,0 +1,62 @@
+|:----------|:------------------------------------------------|:-------------------------|
+| AWS       | CUR                                             | pricing/unit             |
+| GCP       | BigQuery Billing Export                         | usage.pricing_unit       |
+| Microsoft | Cost details                                    | UnitOfMeasure            |
+| OCI       | Cost and Usage Report;<br>List Pricing REST API | cost/billingUnitReadable is closest match<br>*Note: The values in this column are similar but not identical to those in the List Pricing REST API; therefore, it may be preferable to rely on the List Pricing REST API, which we already plan to use for ListUnitPrice.* |
+
+## Example usage scenarios
+
+Current values observed in billing data for various scenarios:
+
+| Provider  | Scenario                | Pattern                                                                                                                              |
+|-----------|-------------------------|--------------------------------------------------------------------------------------------------------------------------------------|
+| AWS       | CUR                     | hours; Hrs; Queries; GB; Secrets; vCPU-Hours; API Requests; Keys; Alarms etc.                                                        |
+| GCP       | BigQuery Billing Export | tebibyte; count; gibibyte; tebibyte; hour; gibibyte hour; gibibyte month; gibibyte hour; gibibyte; hour; count; gibibyte month; etc. |
+| Microsoft | Cost details            | 1 Hour; 10 Hours; 1/Month; 1 GB; 10K; 1 GB-Month; etc.                                                                               |
+| OCI       | Cost and Usage Report   | ONE GiB HOURS STORAGE_SIZE; ONE GiB HOURS MEMORY; etc.                                                                               |
+| OCI       | List Pricing REST API   | Gigabyte Per Hour; etc.<br>*Note: Gigabyte Per Hour corresponds to both ONE GiB HOURS STORAGE_SIZE and ONE GiB HOURS MEMORY*         |
+
+## References and Resources
+
+### AWS
+
+* [Pricing details - AWS Cost and Usage Reports](https://docs.aws.amazon.com/cur/latest/userguide/pricing-columns.html)
+
+### GCP
+
+* [Structure of Detailed data export | Cloud Billing](https://cloud.google.com/billing/docs/how-to/export-data-bigquery-tables/detailed-usage)
+
+### Microsoft
+
+* [Understand usage details fields - Microsoft Cost Management](https://learn.microsoft.com/en-us/azure/cost-management-billing/automate/understand-usage-details-fields)
+
+### OCI
+
+* [Cost and Usage Reports Overview](https://docs.oracle.com/en-us/iaas/Content/Billing/Concepts/usagereportsoverview.htm)
+
+## Discussion / Scratch space
+
+### PricingMeasure - non-normalized vs normalized
+
+* After several discussions the team has decided to use **PricingMeasure (non-normalized)** instead of earlier proposed PricingUnit (semi-normalized).
```

Comment:
I won't hold up the approval of this dimension with this comment, but I do want to be on record recommending against this recent name change.

In reporting and semantic models, "Measure" means something quite different than "Measurement" or "Unit of Measure".  A measure is an aggregation (e.g. sum, average, first, last) of something, typically a number, whereas a measurement or UoM is an attribute which we might slice a measure.

The leading semantic model platforms are quite aligned on the use of the term.  See the following:

[Power BI](https://learn.microsoft.com/en-us/power-bi/transform-model/desktop-measures)
[Tableau](https://help.tableau.com/current/pro/desktop/en-us/datafields_typesandroles.htm)
[Cognos](https://www.ibm.com/docs/en/cognos-analytics/11.1.0?topic=dimensions-creating-measure)
[Qlik](https://help.qlik.com/en-US/cloud-services/Subsystems/Hub/Content/Sense_Hub/Measures/measures.htm)

The name "Pricing Measure" implies that the values contained therein will include something about measures associated with pricing.  The end user will have to remember that this is not the case, and I assume that a non-trivial portion of FOCUS consumers will end up changing this to "Measurement" or "Unit" or "UoM" or "Unit of Measure" in their data models for the purposes of self-documentation anyway.  As a FinOps data modeler, I can assure you that I personally will.

Obviously this name was discussed and agreed, so I won't stop the train from leaving the station.  But I am certain that this name will cause some amount of confusion down the road.

Reply:
For the similar reason, it took me a while to get accustomed to the way we use the term 'Attribute'. :)

Reply:
I absolutely plan to advocate for changing the use of the terms "dimension" and "attribute" post 1.0 to align more with common data modeling parlance... but that's a debate for another day!

Reply:
Just putting it out there as an option to consider. If we wanted to avoid conflict of terms (and I believe we should) `PricingMeasurementUnit` is an option that while is a long name, it feels like a good descriptor for the content and avoids this conflict. Also happy if everyone wants to proceed with `PricingMeasure` as the internet (Chat-GPT) also offers `“measure” refers to a standard unit used to express size, quantity, or degree`

Reply:
Agree that Measure is confusing (used interchangeably with Metric for data analysis). Measurement is even more confusing. 

ChatGPT says: 
"Measure" and "measurement" are related terms, but they have distinct meanings:

Measure:

"Measure" can function as both a verb and a noun.
As a verb, it means to ascertain the size, dimensions, quantity, or degree of something. For example, you might measure the length of a table.
As a noun, "measure" refers to a standard unit used to express size, quantity, or degree. In this sense, it is a reference to a specific dimension or extent. For example, meters or feet are measures of length.
Measurement:

"Measurement" is the process or result of quantifying or determining the extent, size, quantity, or capacity of something using a standardized scale or unit.
It is the act of assigning a number or value to a characteristic or property of an object or phenomenon.
Measurements are typically expressed using specific units, such as inches, kilograms, seconds, etc.
In summary, "measure" is the action of determining the size or quantity, while "measurement" refers to the result or the process of quantifying, often involving the use of standardized units. "Measure" is more action-oriented, while "measurement" is the outcome or the concept of quantification.

Disagree with assessment of dimensions. While in data modeling dimension might mean that, in data analysis, its commonly used to describe qualitative columns that enhance your data and can be used for grouping/filtering etc. Let's revisit post v1.0 :)

Reply:
I respect the fact that ChatGPT thinks "measure" is a possible means of describing a unit in a vacuum, but I will reiterate that every major BI tool that will inevitably be used to house and present FOCUS data has a precise, non-unit-related definition of that specific term.

I prefer `PricingMeasurementUnit` to `PricingMeasure`.  (I do have a separate concern about reserving the obvious choice of `PricingUnit` for a more normative column that will sit alongside this extant column in the future, where the names will not self-document the difference.)  That said, I would prefer any name that sidesteps the specific word "measure".

Reply:
Something to consider...

- NON-NORMALIZED
  - **PricingUnit** (currently named PricingMeasure)
  
- SEMI-NORMALIZED
  - **UsageUnitCategory** (currently named UsageUnit)
  - **PricingUnitCategory** (to be introduced in 1,0+)

Reply:
For me, that (or something like it) would work well, assuming that the list of values in the semi-normalized columns group the values in the non-normalized equivalents.

Reply:
In an effort to get something in, let's go with `PricingMeasurementUnit` for now and continue the discussion in Slack. I suspect there's a better name, but also feel it's best to get something in while we brainstorm.

The current thinking is to create `PricingUnit` as a semi-normalized column like `UsageUnit` post-1.0 and the chances are it would be used more often than the provider-specific string (`PricingMeasurementUnit`), hence trying to stay away from that one.

Reply:
@salpaysenturus  I do not agree that there is ubiquitous agreement on the use and application of "Measure" among the BI platforms out there. There are as many differences as there are similarities. Also this is introducing a term that is used less frequently than unit. Units of Measure sure. Unit Price is a very common term.

Reply:
I am old-school so I prefer to use Wikipedia rather than ChatGPT:

From https://en.wikipedia.org/wiki/Unit_of_measurement:

> A unit of measurement is a definite magnitude of a quantity, defined and adopted by convention or by law, that is used as a standard for measurement of the same quantity.

So `PricingMeasurementUnit `or `PricingUnitOfMeasurement `would be better than `PricingMeasure`.

Reply:
opted for semi-normalized PricingUnit